docs/demo/embed.md

```diff
@@ -28,6 +28,8 @@ Embedding image files as, say, `![[disaster-girl.jpg]]` is equivalent to `![](pa
 
 ![[disaster-girl.jpg]]
 
+It is also posible to add ![[ema-small.png]] as inline paragraphs.
```

Comment:
Do we really need to add a new image file to the repository? Can't we use `favicon.svg`? Shouldn't something like this work?

```markdown
It is also posible to add images inline (example, here's the 
site favicon: [![[favicon.svg]]]{.w-6}) say in the middle of a paragraph.
```

Reply:
Good to know that we can use that image as well.